### PR TITLE
Allow multiple audiences

### PR DESCRIPTION
**Pull request by stang**
See https://github.com/gambol99/keycloak-proxy/issues/400.

### CODE REVIEW

File: user_context_test.go [hunk 1]
```diff
@@ -24,14 +24,17 @@ import (
 
 func TestIsAudience(t *testing.T) {
 	user := &userContext{
-		audience: "test",
+		audiences: ["test", "test2"],
```
Comment (gambol99) [inline user_context_test.go]: missing the `[]string{}`

**Comment by gambol99** [comment]
very much appreciated @stang .. if you can just fix up the tests .. LGTM

File: user_context_test.go (hunk 1 above)
Comment (stang) [reply user_context_test.go]: My bad. It's now fixed.

**Comment by gambol99** [comment]
thank you kindly @stang ....

**Comment by stang** [comment]
cheers @gambol99!

I'll close the related feature request.

**Pull request merged**